### ISSUE TITLE
Widgets Editor: Edit buttons in Block areas inspector move focus unexpectedly

### ISSUE THREAD

**Issue by enriquesanchez** [WordPress/gutenberg#25965]
**Describe the bug**
When pressing one of the 'Edit' block area buttons in the inspector, users are not made aware of what will happen and this can result in disorientation and focus loss, particularly affecting users of assistive technology.

![2020-10-08 13-53-21 2020-10-08 13_57_03](https://user-images.githubusercontent.com/3276087/95509140-b6f2f280-0979-11eb-9670-fead20418aed.gif)

You'll also notice in the gif above, that while selecting 'Footer #2 Edit' takes me to that block area, it is kept in its collapsed state and I still have to <kbd>Tab</kbd> twice in order to now be able to edit it.

**Expected behavior**
I recommend we provide a clearer description to users of AT of what will happen if they select the button ('Click here to move navigate to the Footer #1 block area').

Once the user selects one of these 'Edit' buttons, the block are it navigates to should be expanded by default so users can start editing right away.

I also can't think of any other example where we have a similar navigational pattern in the block editor, so it may also be worth reconsidering why these exist in the first place. It's also very similar in functionality to the 'Outline' menu:

<img width="301" alt="Screen Shot 2020-10-08 at 13 59 19" src="https://user-images.githubusercontent.com/3276087/95509411-194bf300-097a-11eb-8ebb-991dcfa14734.png">

...so I wonder if we can instead merge these two and reduce complexity.

**Comment by talldan** [comment]
#25943 removed the edit buttons, so I think this can be closed now.
